softgroup/services/profile/profile-impl/src/main/java/com/softgroup/profile/impl/handler/GetContactProfilesHandler.java

```diff
@@ -18,13 +28,34 @@
         extends AbstractRequestHandler<GetContactProfilesRequest,
             GetContactProfilesResponse> implements ProfileRequestHandler {
 
+    @Autowired
+    private ContactService contactService;
+
+    @Autowired
+    private TokenService tokenService;
+
     public String getName() {
         return "get_contact_profiles";
     }
 
 
     @Override
     public Response<GetContactProfilesResponse> doHandle(Request<GetContactProfilesRequest> request) {
-        return null;
+        GetContactProfilesRequest requestData = request.getData();
+        GetContactProfilesResponse getContactProfilesResponse = new GetContactProfilesResponse();
+
+        String profileId = request.getRoutingData().getProfileId();
+        List<ContactEntity> contactEntities = contactService.findByProfileId(profileId);
+        getContactProfilesResponse.setContactEntities(contactEntities);
```

Comment:
@VadymPolyanski The flow is wrong. "get_contact_profiles" must provide different response - read API